main.py

```
@@ -86,8 +86,253 @@ async def process_document_template(file: UploadFile = File(...)):
+
+    except Exception as e:
+        return JSONResponse({'status': 'error', 'message': f"Error in parallel processing: {str(e)}"}, status_code=500)
+
+    return JSONResponse({
+        'status': 'success', 
+        'pdf_base64': pdf_base64
+    })
+
+
 @app.post('/api/v1/process-template-document/docx-to-pdf')
 async def process_document_template(data: Dict[str, Any] = Body(...)):
```

Ruff (0.14.1)

Redefinition of unused `process_document_template` from line 74

(F811)